e2e/jenkins/crw-ci/pr-check/k8s/Jenkinsfile

```diff
@@ -14,225 +14,294 @@ pipeline {
-
-                      mkdir -p $LOGS_AND_CONFIGS/che-config
-                      kubectl get configmaps --namespace=che che -o yaml > $LOGS_AND_CONFIGS/che-config/configmap.yaml || true
-                      kubectl get ingresses --namespace=che > $LOGS_AND_CONFIGS/che-config/ingresses.txt || true
-                      kubectl get all -n che -o yaml > $LOGS_AND_CONFIGS/che-config/all.yaml || true
-
-                      mkdir -p $LOGS_AND_CONFIGS/che-logs
-                      kubectl logs --namespace=che \$(kubectl get pods --namespace=che | grep che- | awk {'print \$1'})>$LOGS_AND_CONFIGS/che-logs/che-server.log || true
-                      kubectl logs --namespace=che \$(kubectl get pods --namespace=che | grep devfile-registry- | awk {'print \$1'})>$LOGS_AND_CONFIGS/che-logs/devfile-registry.log || true
-                      kubectl logs --namespace=che \$(kubectl get pods --namespace=che | grep plugin-registry- | awk {'print \$1'})>$LOGS_AND_CONFIGS/che-logs/plugin-registry.log || true
-                        
-                      mkdir -p $LOGS_AND_CONFIGS/workspace-logs
-                      export WS_POD=\$(kubectl get pod --namespace=che | grep  workspace | awk '{print \$1}')
-                      for c in \$(kubectl get pod --namespace=che \$WS_POD -o jsonpath="{.spec.containers[*].name}") ; do kubectl logs \$(kubectl get pod --namespace=che | grep  workspace | awk '{print \$1}') "\${c}" -n che > $LOGS_AND_CONFIGS/workspace-logs/"\${c}"-container.log || true; done
-                      kubectl describe pod --namespace=che \$WS_POD > $LOGS_AND_CONFIGS/che-config/workspace-pod-description.txt || true
-                """
+            sh """                      
+                mkdir -p $LOGS_AND_CONFIGS/kubectl
+                kubectl --namespace=che get events > $LOGS_AND_CONFIGS/kubectl/events.txt || true
 
-                sh """
-                      /usr/local/bin/minikube stop || true
-                    
-                      sudo umount \$(mount | grep "kubelet" | awk '{if(NR>0) print \$3}') || true
-                      sudo rm -rf `sudo find /tmp -name 'hostpath-provisioner' 2>/dev/null` || true
-                    
-                      docker volume rm \$(docker volume ls -q -f dangling=true) || true
-                      
-                      docker rm -f \$(sudo docker ps --all | awk 'NR>0 {print \$1;}') || true
-                      
-                      if [[ \$(docker images | grep '${CHE_IMAGE_REPO}') != "" ]]; then
-                        docker rmi -f \$(docker images | grep '${CHE_IMAGE_REPO}') || true
-                      fi
-                """
+                mkdir -p $LOGS_AND_CONFIGS/che-config
 
-                archiveArtifacts allowEmptyArchive: true, artifacts: "e2e/report/**, logs-and-configs/**"
+                cp ${WORKSPACE}/org_v1_che_cr.yaml $LOGS_AND_CONFIGS/che-config || true
```

Comment:
Wouldn't it be better to add `set +e` and `set -e` for a block of commands than adding ` || true` after each command?

Reply:
Yes, it would be better, by I don't now how to do it in pipeline so that it works for ALL sh commands.

Reply:
I see, I'm not much familiar with pipelines, but maybe using this `set` in each script when more commands are running with `|| true` will help too. But I'm OK event with having it as it is. :)